db/seeds/003_users.rb

```diff
@@ -0,0 +1,8 @@
+# db/seeds.rb
+exit unless Rails.env.development?
+
+admin = User.find_or_initialize_by(email: 'admin@example.com')
+admin.password = 'password'
+admin.save!(validate: false)
```

Comment:
I think you can condense this down to one line:
```suggestion
User.find_or_initialize_by(email: 'admin@example.com').update(password: 'password')
```